mentors/r/r.json

```diff
@@ -1 +1,10 @@
-[]
+[
+  {
+  "github_username": "rrrlw",
```

Comment:
Can you please indent fields another two spaces?

Reply:
Fixed in commit 3617c2b